WTWSMS/decisions/realm_decisions.txt

```diff
@@ -2028,26 +2061,6 @@ decisions = {
 					value = off
 				}
 			}
-			
-			OR = {
-				ai = no
-				AND = {
-					trait = proud
-					prestige = 5000
-				}
-				AND = {
-					trait = ambitious
-					prestige = 5000
-				}
-				AND = {
-					realm_size = 70
-					any_demesne_title = {
-						count = 4
-						tier = DUKE
-					}
-				}
-				prestige = 15000
-			}
```

Comment:
Rulers didn't form that many kingdoms historically. It should be a bit rare and exclusive. Completely removing all restrictions is nonsense here, that would mean every statelet would proclaim they are the one true Roman Empire every other day. We are going to add the dynamic dejure too, by the way. All in all it has to have some restrictions to make sense historically. They can be loosened, but not removed altogether. Empires were extremely rare. When you referred to this previously, I understood it as it was for the player, hence my incomprehension.